armada/handlers/chartbuilder.py

```diff
@@ -62,13 +62,41 @@ def __init__(self, chart, parent=None):
         # extract, pull, whatever the chart from its source
         self.source_directory = self.get_source_path()
 
+        # load ignored files from .helmignore if present
+        self.ignored_files = self.get_ignored_files()
+
     def get_source_path(self):
         '''
         Return the joined path of the source directory and subpath
         '''
         return os.path.join(self.chart.source_dir[0],
                             self.chart.source_dir[1])
 
+    def get_ignored_files(self):
+        '''
+        Load files from .helmignore if present
+        '''
+        ignored_files = []
+        if os.path.exists(os.path.join(self.source_directory, '.helmignore')):
+            with open(os.path.join(self.source_directory, '.helmignore')) as f:
+                ignored_files = f.readlines()
+        return [filename.strip() for filename in ignored_files]
+
+    def ignore_file(self, filename):
+        '''
+        :params file - filename to compare against list of ignored files
+
+        Returns true if file matches an ignored file wildcard or exact name,
+         false otherwise
+        '''
+        for ignored_file in self.ignored_files:
+            if (ignored_file.startswith('*') and
```

Comment:
we could. seems like an unnecessary import for a simple pattern but the code would probably be cleaner